python/doc/theory/data_analysis/kolmogorov_test.rst

```diff
@@ -1,87 +1,55 @@
 .. _kolmogorov_smirnov_test:
 
-Kolmogorov-Smirnov fitting test
--------------------------------
+Kolmogorov-Smirnov test
```

Comment:
There are 2 pages in the doc called "Kolmogorov-Smirnov test": `kolmogorov_test.rst` and `smirnov_test.rst` depending on whether 2 samples are tested or a sample and a distribution. Does it make sense to have this separation in the doc? Note that this is not done that way in `graphical_fitting_test.rst` where both use cases are presented.

Reply:
I have seen that point. I guess that we wanted to help the User and lead him straight to the right section very quickly. I have seen that the graphical tests gather all the situations.
I propose to leave it like it is. It is not a problem.

Reply:
I agree, I was just wondering.

Reply:
I agree with Joseph: there cannot be two pages with the same title. Here is the title given in [^Ross2004] page 504.
```suggestion
The Kolmogorov-Smirnov goodness of fit test for continuous data
```

Please don't forget to extend the underline string below so that it matches the new length.

[^Ross2004]: Ross, S. (2004). Probability and statistics for engineers and scientists. Third Edition. Elsevier.

Reply:
I guess this is settled then, you may resolve this conversation.